src/content/documentation/publish/version-compatibility.md

```diff
@@ -0,0 +1,161 @@
+
+The [`browser_specific_settings`](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/browser_specific_settings) manifest key allows you to statically declare what versions of Firefox can load the extension. Firefox versions outside of the supported range will fail during the extension's installation process.
+
+Since these values are hardcoded in the extension itself, they cannot be changed after a version has been packaged. When distributing on AMO, you must submit a new version of the add-on to modify the range of browsers that can load the extension
+
+There are two relevant subkeys for Firefox browsers:
+
+| Key             | Description                                                  |
+| --------------- | ------------------------------------------------------------ |
+| `gecko`         | Used to specify settings for the desktop version of Firefox. |
+| `gecko_android` | Specifies settings for the Android version of Firefox.       |
+
+::: note
+Firefox for iOS and Firefox Focus do not support add-ons and therefore do not have subkeys.
+:::
+
+Both `gecko` and `gecko_android` accept two version-related subkeys:
+
+| Subkey               | Description                                                                                                       |
+| -------------------- | ----------------------------------------------------------------------------------------------------------------- |
+| `strict_min_version` | Specifies the lowest version of the browser that can load the extension.                                          |
+| `strict_max_version` | Specifies the highest version of the browser that can load the extension.<br>*Avoid unless absolutely necessary.* |
+
+### Android compatibility
+
+In order to signal that your extension is compatible with Android, your manifest must at least set `browser_specific_settings.gecko_android` to an empty object. If you don't, AMO will assume that the extension is not compatible with Android and will not list it as available on Android. This can be manually overridden using AMO's compatibility controls. Omitting `"gecko_android"` does not affect Firefox for Android's ability to install the extension.
+
+```json
+{
+  "name": "Desktop & Android extension",
+  "version": "1.0",
+  "manifest_version": 2,
+  "browser_specific_settings": {
+    "gecko_android": {}
+  }
+}
+```
+
+{% endcapture %}
+{% include modules/one-column.liquid,
+  id: "browser-specific-settings"
+  content: content
+%}
+
+<!-- END: Content with Table of Contents Module -->
+
+<!-- Single Column Body Module -->
+
+{% capture content %}
+
+## AMO compatibility setting
+
+AMO's browser version compatibility settings determine which versions of Firefox can search for and install a given extension. The browser compatibility settings for a newly uploaded version of an extension are automatically by the `browser_specific_settings` of the extension's manifest.
+
```

Comment:
@dotproto don't we need something here to explain why you would change the settings? I can only think of something along the lines of "For example, if you discover that your extension doesn't work on the latest version of Firefox due to a regression or similar issue in the web extension API, you can limit the maximum version that your extension install is on until the issue is fixed. "

Also, a note along the lines of "Remember that, if you set a wider range of supported versions in AMO, your extension may list in searches but not installed due to the Firefox version limits set in its manifest file."

Reply:
+1 to both of those suggestions. I'll integrate locally & push and update soon.